src/partition/repart.c

```diff
@@ -1313,12 +1314,50 @@ static int config_parse_gpt_flags(
         return 0;
 }
 
+static int config_parse_uuid(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        Partition *partition = ASSERT_PTR(data);
+        int r;
+
+        if (isempty(rvalue)) {
+                partition->new_uuid = SD_ID128_NULL;
+                partition->new_uuid_is_set = false;
+                return 0;
+        }
+
+        if (streq(rvalue, "empty")) {
```

Comment:
Is "empty" the right word? we can't really set it to "empty". only to `00000000-0000-0000-0000-000000000000`, which maybe we should call "null" (after all, the C macro for it is called `SD_ID128_NULL`?)

Also, please mention this shortcut in the man page